README.md

```diff
@@ -1,24 +1,48 @@
-# README
+# Junk4Dollars
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+[![Build Status](https://travis-ci.com/riccjohn/junk4dollars-api.svg?branch=master)](https://travis-ci.com/riccjohn/junk4dollars-api)
 
-Things you may want to cover:
+# Installation
 
-* Ruby version
+## Rails
 
-* System dependencies
+Install `rails 6.0.0`
 
-* Configuration
+```zsh
+gem install rails -v 6.0.0
+```
 
-* Database creation
+Clone this repo
 
-* Database initialization
+```zsh
+git clone git@github.com:riccjohn/junk4dollars-api.git
+```
 
-* How to run the test suite
+Install dependencies
 
-* Services (job queues, cache servers, search engines, etc.)
+```zsh
+cd junk4dollars-api && bundle install
+```
 
-* Deployment instructions
+## Starting the server locally
```

Comment:
I feel like some steps are missing. Does there need to be explicit steps to create and/or migrate the database?